opencti-platform/opencti-front/lang/back/en.json

```diff
@@ -271,6 +272,7 @@
   "Feed types": "Feed types",
   "File errors": "File errors",
   "File ID": "File ID",
+  "File id": "File id",
```

Comment:
maybe we could use existing "File ID" translation ? instead of creating a new one